stuhlandia.py

```diff
@@ -0,0 +1,21 @@
+row1 = set('qwertyuiop\n')
+row2 = set('asdfghjkl\n')
+row3 = set('zxcvbnm\n')
+
+try:
+    with open("states.txt", "r") as f:
+        state = f.readline()
+        while state:
+            start_set = set(state)
+            if len(set(start_set.difference(row1))) == 0:
+                print(f'In row 1: {state}')
+                break
```

Comment:
You should avoid using breaks if you can, when you use funcitons you can just return the value